client/src/components/Accordions.jsx

```diff
@@ -1,10 +1,69 @@
 import React from 'react';
-import {AccordionContainer} from '../styling.jsx';
+import styled from 'styled-components';
+import AccordionTab from './AccordionTab.jsx';
 
-const Accordions = (props) => {
-  return (
-    <AccordionContainer>Accordions go here</ AccordionContainer>
-  );
+const AccordionContainer = styled.div`
+  padding: 0;
+  background: rgb(250,250,250);
+  margin: 0px 0px 30px;
+  margin-top: 1.5rem;
+  // border: 1px solid rgba(0, 0, 0, .1);
+`;
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+       openedTab: null,
+    }
+    this.updateTabDisplay = this.updateTabDisplay.bind(this);
+  }
+
+  updateTabDisplay(event) {
+    console.log(event.target.innerHTML)
+    this.setState({ openedTab: event.target.innerHTML })
+  }
+
+  render() {
+    const tabs = [
+      {
+        id: 1,
+        title: 'Fabric',
+        content: this.props.fabric,
+      },
+      {
+        id: 2,
+        title: 'Care',
+        content: this.props.care,
+      },
+      {
+        id: 3,
+        title: 'Features',
+        content: 'this is where features goes',
+      }
+    ];
+
+    return (
+      <AccordionContainer>
+        {tabs.map((tab, index) => {
+          return (
+            <AccordionTab
+              openedTab={this.state.openedTab}
+              updateTabDisplay={this.updateTabDisplay}
+              key={tab.id}
```

Comment:
I think that more unique keys are recommended, ie. unique throughout the entire app

Reply:
Would you recommend concatenating "accordion" to the id then?